src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -599,7 +599,7 @@ protected override void EndProcessing()
         // We now ship .Net Core's reference assemblies with PowerShell, so that Add-Type can work
         // in a predictable way and won't be broken when we move to newer version of .NET Core.
         // The reference assemblies are located at '$PSHOME\ref'.
-        private static readonly string s_netcoreAppRefFolder = PathType.Combine(PathType.GetDirectoryName(typeof(PSObject).Assembly.Location), "ref");
+        private static readonly string s_netcoreAppRefFolder = PathType.Combine(PathType.GetDirectoryName(Assembly.GetEntryAssembly().Location), "ref");
```

Comment:
Could you please clarify why we need the change? PSObject is in SMA, Add-Type is in Microsoft.PowerShell.Utility.dll.
 Should we update comment above too?